packages/orchestrator-ui-components/src/pages/processes/WfoStartProcessPage.tsx

```diff
@@ -97,30 +96,15 @@ export const WfoStartProcessPage = ({
     const { getStepHeaderStyle, stepListContentBoldTextStyle } =
         getStyles(theme);
 
-    const { data, isLoading, isError } = useQueryWithGraphql(
-        GET_PROCESS_STEPS_GRAPHQL_QUERY,
-        {
-            processName,
-        },
-        `processSteps={processName}`,
-    );
+    const { data: timeLineItems = [], isError } =
+        useGetTimeLineItemsQuery(processName);
 
     if (isError) {
         if (!hasError) {
             setHasError(true);
         }
     }
 
-    const timeLineItems: TimelineItem[] =
-        data?.workflows?.page[0]?.steps && !isLoading
```

Comment:
Probably slightly related to my comment in `packages/orchestrator-ui-components/src/rtk/endpoints/processSteps.ts`, you might want to act on isLoading from the useGetTimeLineItemsQuery hook

Reply:
Fixed